rfps/smart-nomination-contract.md

```diff
@@ -0,0 +1,98 @@
+# Validator selection smart contract
+
+* **Status:** Open
+* **Proposer:** [mmagician](https://github.com/mmagician)
+
+
+## Project Description :page_facing_up: 
+
+Nomination is not such a trivial task as it may seem. It requires an active effort on the part of the nominators in order to maximise both the network security as well as their own rewards.
+
+There are a couple of reasons for this:
+- new validators joining the ecosystem might have something better to offer
+- validators from the previous active set might no longer make it to the set in the next session
+- validators leaving the ecosystem
+- previous validators degrading their performance
+- validators increasing their commission
+- validators actively blacklisting nominators
+
+In an attempt to tackle these challenges, we propose a smart contract which automatically distributes the nominator's stakes to the best validators, based on the qualitative preferences of the user.
+
+### Preferences
+Different users might care about different criteria when selecing for their validators, for example:
+- comission
+- performance in the last X sessions
+- inclusion in the active set in the last X sessions
+- total number of sessions active
+- slashing history
+- etc.
```

Comment:
These are mostly meaningless from a security perspective.  We'd likely remove nominators entirely if these were meaningful.